<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1644</width>
    <height>1164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="howmany">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>1훈솜사탕 R</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Where do you want to go?</string>
    </property>
   </widget>
   <widget class="QLabel" name="home">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>1훈솜사탕 R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="subway_map.qrc">:/home/cabin.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_picture">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>1641</width>
      <height>971</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="subway_map.qrc">:/subwaymap/subwaymap.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>850</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="topbgcolor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1661</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="topbgcolor_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1010</y>
      <width>1671</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_snowman">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="subway_map.qrc">:/subwaymap/snowman.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="people_group_22">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>1훈솜사탕 R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>TO</string>
    </property>
    <widget class="QLineEdit" name="p_start_lbl_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Regular</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="people_group_11">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>1훈솜사탕 R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>From</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="p_start_lbl_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Regular</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="run_button3">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>20</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>topbgcolor_2</zorder>
   <zorder>topbgcolor</zorder>
   <zorder>howmany</zorder>
   <zorder>home</zorder>
   <zorder>label_picture</zorder>
   <zorder>label</zorder>
   <zorder>label_snowman</zorder>
   <zorder>people_group_22</zorder>
   <zorder>run_button3</zorder>
   <zorder>people_group_11</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1644</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="subway_map.qrc"/>
 </resources>
 <connections/>
</ui>
